include/fiction/io/print_layout.hpp

```diff
@@ -472,7 +473,7 @@ void print_sidb_layout(std::ostream& os, const Lyt& lyt, const bool cs_color = t
 
         if (!already_printed && lyt.get_cell_type(loop_coordinate) != sidb_technology::cell_type::EMPTY)
         {
-            os << fmt::format(cs_color ? detail::SIDB_DEF_NEU_COLOR : detail::NO_COLOR, " o ");
+            os << fmt::format(cs_color ? detail::SIDB_DEF_NEU_COLOR : detail::NO_COLOR, " ◯ ");
```

Comment:
Maybe we can just always use ``detail::SIDB_LAT_COLOR``, right?

Reply:
You mean ignoring the `cs_color` flag?

Reply:
I've renamed the flag to better convey its usage. It is supposed to toggle all color escapes such that users may disable color output completely in case it breaks their terminal, which happens...

Reply:
ah, perfect! I did not know that it can cause issues.